docs/snippets/security-prereqs-images.md

```diff
@@ -0,0 +1,26 @@
+## Verifying Images
```

Comment:
```suggestion
## Verifying image signatures
```

Please ensure that sentence case is used for titles, per the https://github.com/knative/docs/blob/main/contribute-to-docs/style-guide/style-and-formatting.md